tests/Humbug/Test/Phpunit/Logger/JsonLoggerTest.php

```diff
@@ -22,9 +25,8 @@ public function testShouldThrowExceptionWhenTargetIsNotSpecified()
 
     public function testShouldWriteLogsDuringDestruct()
     {
-        $jsonLogger = $this->getMock('\Humbug\Phpunit\Logger\JsonLogger', ['write'], ['/target/path']);
-
-        $jsonLogger->expects($this->once())->method('write');
+        $jsonLogger = m::mock("\\Humbug\\Phpunit\\Logger\\JsonLogger")->makePartial();
```

Comment:
We'll survive until this version is done with, we'll need a 2.0 probably for PHPUnit 6 and then it will be ::class all the way!